### PR TITLE
Feature/talentlyft

### PR DESCRIPTION
**Pull request by iruzevic**
### Added

- New TalentLyft integration.

### CODE REVIEW

File: src/Blocks/custom/talentlyft/manifest.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	"description" : "Talentlyft job application form",
+	"category": "eightshift-forms",
+	"icon": {
+		"src": "esf-form-talentlyft"
+	},
+	"keywords": [
+		"talentlyft"
+	],
+	"example": {
+	},
```
Comment (piqusy) [inline src/Blocks/custom/talentlyft/manifest.json:14]: Can be removed?

File: src/Integrations/Talentlyft/SettingsTalentlyft.php [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+					[
+						'component' => 'tab',
+						'tabLabel' => \__('Options', 'eightshift-forms'),
+						'tabContent' => [
+							...$this->getGlobalGeneralSettings(self::SETTINGS_TYPE_KEY),
+							[
+								'component' => 'input',
+								'inputName' => UtilsSettingsHelper::getOptionName(self::SETTINGS_TALENTLYFT_FILE_UPLOAD_LIMIT_KEY),
+								'inputFieldLabel' => \__('Max upload file size', 'eightshift-forms'),
+								'inputFieldHelp' => \__('Up to 5MB.', 'eightshift-forms'),
```
Comment (piqusy) [inline src/Integrations/Talentlyft/SettingsTalentlyft.php:256]: Can the value of `self::SETTINGS_TALENTLYFT_FILE_UPLOAD_LIMIT_KEY` be `sprint_f`...ed here? 😄

**Review by goranalkovic-infinum — DISMISSED** [review]

File: src/Integrations/Talentlyft/TalentlyftClient.php [hunk 3]
```diff
@@ -0,0 +1,512 @@
...
+	 */
+	public function __construct(EnrichmentInterface $enrichment)
+	{
+		$this->enrichment = $enrichment;
+	}
+
+	/**
+	 * Return items.
+	 *
+	 * @param bool $hideUpdateTime Determin if update time will be in the output or not.
```
Comment (piqusy) [inline src/Integrations/Talentlyft/TalentlyftClient.php:55]:
```suggestion
	 * @param bool $hideUpdateTime Determine if update time will be in the output or not.
```

File: src/Rest/Routes/Integrations/Talentlyft/IntegrationItemsTalentlyftRoute.php [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+			'request' => $request,
+		];
+
+		// Check if Talentlyft global settings is valid.
+		$isGlobalSettingsValid = \apply_filters(SettingsTalentlyft::FILTER_SETTINGS_GLOBAL_NAME, false);
+
+		if (!$isGlobalSettingsValid) {
+			return \rest_ensure_response(
+				UtilsApiHelper::getApiErrorPublicOutput(
+					\esc_html__('Global not configured', 'eightshift-forms'),
```
Comment (piqusy) [inline src/Rest/Routes/Integrations/Talentlyft/IntegrationItemsTalentlyftRoute.php:93]:
```suggestion
					\esc_html__('Global settings not configured', 'eightshift-forms'),
```

**Review by piqusy — CHANGES_REQUESTED** [review]
Nice work 👏🏼 

Left a few suggestions

**Review by goranalkovic-infinum — APPROVED** [review]

**Review by piqusy — APPROVED** [review]

**Pull request merged**